src/Verify.MSTest/TestContextReflector.cs

```diff
@@ -1,129 +0,0 @@
-namespace VerifyMSTest;
-
-static class TestContextReflector
-{
-    const BindingFlags ReflectionBindingFlags = BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic;
-    static readonly ConcurrentDictionary<string, Type?> TypeCache = new();
-
-    public static (Assembly Assembly, Type Type, MethodInfo Method) Get(TestContext context)
-    {
-        var testName = context.TestName;
-        var typeName = context.FullyQualifiedTestClassName!; // Checked for null in the Verifier
-
-        if (!TryGetTypeFromTestContext(typeName, context, out var type))
-        {
-            type = FindType(typeName);
-        }
-
-        var testNameSpan = testName.AsSpan();
```

Comment:
@SimonCropp, I don't think this code is needed anymore; let me know if you disagree.

Before this refactoring, no tests failed when I commented out this substring code. Doing a `git blame` back in time points to f4b31d2, but it's unclear to me what the previous behavior.

If you know what this code is for / is still needed let me know and I'll add a test. Thanks!

Reply:
yeah thats a shitty change by me. lets assume for now it is no longer required.  i will advertise the beta and hopefully if it is required, it will become apparent why